### PR TITLE
Fix spelling of name

### PR DESCRIPTION
**Pull request by gavanderhoorn**
Corrected my name.

### CODE REVIEW

File: source/existing/09_vanderhoorn.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
-.. _Gijs Vanderhoorn:
+.. _Gijs van der Hoorn:
```
Comment (gavanderhoorn) [inline source/existing/09_vanderhoorn.rst:1]: Not sure this is correct Sphinx syntax.

File: source/existing/09_vanderhoorn.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-Gijs Vanderhoorn
+Gijs van der Hoorn
 ================
```
Comment (fmauch) [inline source/existing/09_vanderhoorn.rst]:
```suggestion
Gijs van der Hoorn
==================
```

File: source/existing/09_vanderhoorn.rst (hunk 1 above)
Comment (fmauch) [reply source/existing/09_vanderhoorn.rst:1]: Yes, this is fine, but we actually have a reference to that label inside proposal/index.rst which needs to be altered, as well. I seem to be too stupid to link individual lines from the RST document, though...

**Comment by gavanderhoorn** [comment]
Is something more needed for this @fmauch?

**Comment by fmauch** [comment]
> Is something more needed for this @fmauch?

As mentioned in https://github.com/fzi-forschungszentrum-informatik/fzi_robot_interface_proposal/pull/1#discussion_r415778645 there's a broken reference from this change in https://github.com/fzi-forschungszentrum-informatik/fzi_robot_interface_proposal/blob/master/source/proposal/index.rst Line 55 that has to be altered as well.

**Comment by fmauch** [comment]
@gavanderhoorn I took the liberty of adding a commit to your PR. If it's ok with you, I would like to merge it like this.

**Review by fmauch — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
> @gavanderhoorn I took the liberty of adding a commit to your PR. If it's ok with you, I would like to merge it like this.

Apologies for taking so long here.

👍 from me

**Pull request merged**